backend/services/implementations/userService.ts

```diff
@@ -638,6 +666,30 @@ class UserService implements IUserService {
   ): Promise<VolunteerUserResponseDTO> {
     let firebaseUser: firebaseAdmin.auth.UserRecord;
 
+    try {
+      const userInvite = await prisma.userInvite.findUnique({
+        where: {
+          uuid: volunteerUser.token,
+        },
+      });
+
+      if (userInvite == null) {
```

Comment:
[extreme nitpick] but ??? 
```suggestion
      if (userInvite === null) {
```